### PR TITLE
fix(server): Fix outcomes endpoint url

### PR DESCRIPTION
**Pull request by RaduW**
Fix bug that was preventing the relays to forward outcomes from downstream relays.

### CODE REVIEW

File: tests/integration/test_outcome.py [hunk 1]
```diff
@@ -276,3 +287,94 @@ def test_outcome_source(relay, mini_sentry):
     outcome = outcomes[0]
 
     assert outcome.get("source") == "my-layer"
+
+
+def test_outcome_forwarding(relay, mini_sentry):
```
Comment (jan-auer) [inline tests/integration/test_outcome.py]: For testing all these cases, you can just use `relay_chain`, like here:

https://github.com/getsentry/relay/blob/fea4f20c8be5b6a54153cde358bf901d3df6c7ef/tests/integration/test_envelope.py#L4-L5

This is a parameterized fixture that will create multiple tests with variable length chains and also HA proxies between them.

Comment (RaduW) [reply tests/integration/test_outcome.py]: yes I know the relay_chain... I wanted to explicitly control what relays I use and control their configuration so I can figure out which one is the source of the outcome

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ We have switched to [CalVer](https://calver.org/)! Relay's version is always in
 - Respond with `403 Forbidden` when multiple authentication payloads are sent by the SDK. Previously, Relay would authenticate using one of the payloads and silently ignore the rest. ([#602](https://github.com/getsentry/relay/pull/602))
 - Improve metrics documentation. ([#614](https://github.com/getsentry/relay/pull/614))
 - Do not scrub event processing errors by default. ([#619](https://github.com/getsentry/relay/pull/619))
+- Outcomes from downstream relays were not forwarded upstream. ([#632](https://github.com/getsentry/relay/pull/632))
```
Comment (jan-auer) [inline CHANGELOG.md]: `20.6.0` is already released. Please add to a new "Unreleased" section.

**Review by jan-auer — CHANGES_REQUESTED** [review]
Looks good. Can be merged once the changelog is fixed.

**Review by jan-auer — APPROVED** [review]

**Pull request merged**